### PR TITLE
Add "FastNoiseSIMD" package

### PR DESCRIPTION
**Pull request by caseymcc**
* I've followed [this guide](https://docs.hunter.sh/en/latest/creating-new/create/cmake.html)
  step by step carefully. Yes

* I've tested this package remotely and have excluded all broken builds.
  Here is the links to the Travis/AppVeyor with status "All passed":

  * https://ci.appveyor.com/project/caseymcc/hunter/builds/23957512
  * https://travis-ci.org/caseymcc/hunter/builds/521892308

* I have submitted CI configs to https://github.com/ingenue/hunter targeting `pkg.template` branch,
  see this merged pull request https://github.com/ingenue/hunter/pull/356

### CODE REVIEW

File: docs/packages/pkg/FastNoiseSIMD.rst [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+.. index:: math ; FastNoiseSIMD
+
+.. _pkg.FastNoiseSIMD:
+
+FastNoiseSIMD
+=============
+
+-  `Original <https://github.com/Auburns/FastNoiseSIMD>`__
+-  `Official <https://github.com/caseymcc/FastNoiseSIMD>`__
```
Comment (ruslo) [inline docs/packages/pkg/FastNoiseSIMD.rst]: Can you explain the original/official part? Do you want me to create https://github.com/hunter-packages fork?

Comment (caseymcc) [reply docs/packages/pkg/FastNoiseSIMD.rst]: The original was a macro base SIMD version, I modified it heavily to templates and altered its interface. I believe the original version is now defunct.

Comment (ruslo) [reply docs/packages/pkg/FastNoiseSIMD.rst]: > I modified it heavily to templates and altered its interface

So they are not compatible? Would it be better to rename the project and introduce completely new API? You can leave the "Originally based on ... work" note somewhere.

**Comment by caseymcc** [comment]
Yeah not compatible, I'll just remove the original info it's there in
github anyway. Don't really want to change the name though as it it will
work like the original with a few just a few modifications.

**Comment by ruslo** [comment]
> Yeah not compatible, I'll just remove the original info it's there in github anyway. Don't really want to change the name though as it it will work like the original with a few just a few modifications.

That is pretty strange from the user point of view. You hack the API, you're not planning to send your patches upstream, you haven't changed the name of the project and just set it here as "official".

**Comment by caseymcc** [comment]
I did send the changes up stream about 6 months ago, no word.

However I will go ahead and change the name to something else.

**Comment by ruslo** [comment]
> I did send the changes up stream about 6 months ago, no word.

Hm, yeah I don't see much activity there. Have you tried to contact the author https://github.com/Auburns ?

**Comment by caseymcc** [comment]
He works for "No Man's Sky"(https://www.nomanssky.com/) now, I am guessing this kind of open source work is off limits for him now. I will just re-name it and re-upload. Ill go ahead and close this PR and send a new one when I get the changes made. You will likely need to revert the pkg.fastnoisesimd merge.

**Comment by ruslo** [comment]
> I am guessing this kind of open source work is off limits for him now

My point was: if he is not planning to maintain the project you can ask him to give you such a role. Just adding a note to the README about your fork and pressing "Archive this repository" is not something that will require a lot of time.

> I will just re-name it and re-upload

It sounds good for me too.

> You will likely need to revert the pkg.fastnoisesimd merge.

Ok, I will remove `pkg.fastnoisesimd` branch.

**Pull request closed without merging**